src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesignerIntermediateClass.cs

```diff
@@ -21,13 +21,15 @@ public class GenerateResourceDesignerIntermediateClass : AndroidTask
 // </auto-generated>
 //------------------------------------------------------------------------------
 using System;
+using System.CodeDom.Compiler;
 
 namespace %NAMESPACE% {
 	#pragma warning disable IDE0002
 	/// <summary>
 	/// Android Resource Designer class.
 	/// Exposes the Android Resource designer assembly into the project Namespace.
 	/// </summary>
+	[GeneratedCode(""%TOOL%"", ""%VERSION%"")]
```

Comment:
Does this need to do:
```suggestion
	[global::System.CodeDom.Compiler.GeneratedCode(""%TOOL%"", ""%VERSION%"")]
```
I'm just thinking if someone makes their own attribute named `GeneratedCode`.